src/libs/ajax/DatasetBuilder.ts

```diff
@@ -31,16 +33,25 @@ export interface ProgramDataListOption extends ProgramDataOption {
 
 export interface DatasetResponse {
   name: string;
+  id: string;
+  description: string;
```

Comment:
We should implement this as being markdown. It can be done now or in a future PR

Reply:
Hmm… I think string or string[] is how you represent markdown as a type in the frontend - that's the type on the `MarkdownViewer` props at least